### PR TITLE
OPSEXP-2462 Create new values file for pre-released versions and make default values file only for released images

### PR DESCRIPTION
**Pull request by pmacius**
!Ref OPSEXP-2462

### CODE REVIEW

**Comment by pmacius** [comment]
Please take a look if this looks all right, then I will test pipeline bumping versions :)

**Review by alxgomz — CHANGES_REQUESTED** [review]
version based values & docker files should not contain pre-release versions. Was the updatecli pipelines hanged on purpose while we wait for releases to happen?

File: docker-compose/pre-release-docker-compose.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 version: "2"
 services:
   alfresco:
-    image: quay.io/alfresco/alfresco-content-repository:23.1.1
+    image: quay.io/alfresco/alfresco-content-repository:23.2.0-M1
```
[the remark this answers is not in the document]
Comment (alxgomz) [reply docker-compose/pre-release-docker-compose.yml:21]: Argh sorry my bad! I missed the file was renamed

**Comment by pmacius** [comment]
> Was the updatecli pipelines hanged on purpose while we wait for releases to happen?
Yes, I think that it will cause less trouble if we stick to those tags for now and wait for new release.

File: helm/alfresco-content-services/values.yaml
Comment (gionn) [inline helm/alfresco-content-services/values.yaml:1]: would expect tag changes in this file (from -A to GA versions)

**Review by gionn — CHANGES_REQUESTED** [review]

**Comment by pmacius** [comment]
https://github.com/Alfresco/acs-deployment/actions/runs/8082979440/job/22084993920

File: updatecli-matrix-targets.yaml [hunk 2]
```diff
@@ -9,13 +9,13 @@ matrix:
...
-        docker-compose/docker-compose.yml
+        docker-compose/pre-release-docker-compose.yml
       compose_key: $.services.alfresco.image
       helm_target: &helmvaluesNext >-
-        helm/alfresco-content-services/values.yaml
+        helm/alfresco-content-services/pre-release_values.yaml
       helm_key: $.alfresco-repository.image.tag
       # keep the unused anchor to detect multiple usages when copying blocks for new versions
-      helm_update_appVersion: &helm_update_appVersion true
+      helm_update_appVersion: &helm_update_appVersion false
```
Comment (gionn) [inline updatecli-matrix-targets.yaml:18]: remove it, default is false

put this block to `current` (I can explain better why this is needed in private xD)
```
      # keep the unused anchor to detect multiple usages when copying blocks for new versions
      helm_update_appVersion: &helm_update_appVersion true
```

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]

**Review by alxgomz — APPROVED** [review]

**Pull request merged**